index.html

```diff
@@ -2653,7 +2653,7 @@ <h3 id="applying">
               </li>
             </ol>
           </li>
-          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
+          <li><a>Trim</a>(<var>value</var>) and convert <a>to ASCII lowercase</a></li>
```

Comment:
I think this is not a fault of your patch, so I filed bugs instead.

Could you look at the & issue and then maybe we can go ahead and merge?